RELEASING.md

```diff
@@ -1,9 +1,29 @@
 # Releasing
 
+## Pre-release
+
+Before each release, we will publish a Snapshot version of it and use it to create a pre-release on [Github Release page](https://github.com/adevinta/spark-android/releases)
+
+During a period of ~2 weeks, consumers will be able to test it and report bugs that can be fixed before it’s considered stable.
+
+Using the Github pre-release feature will allow Spark users to be notified that a new release is being prepared and test against it.
+For example, they could set up hooks to post the changelog in their monitoring Slack channel or trigger a CI build to validate that this new release doesn’t break their build.
+
+1. [Create a draft release](https://github.com/adevinta/spark-android/releases/new?tag=X.Y.Z-SNAPSHOT&title=X.Y.Z-SNAPSHOT&prerelease=1) with a `*.*.*-SNAPSHOT` version tag.
+2. Click on `Generate release notes` to automatically add all the merged pull requests from this diff and contributors of this release.
+3. Remove logs from `@dependabot` except if they mention big version upgrades for libraries used by our consumers (like Compose or Kotlin). 
+4. Reformat the changelog to be as close as possible to the format we describe in the [CHANGELOG STYLE GUIDE](./docs/CHANGELOG%20STYLE%20GUIDE.md).
+5. If we’re satisfied with the draft, release it but make sure **`⚠️ Set as a pre-release`** is checked.
+6. Wait **at least 2 weeks** for feedback from consumers on the stability of this release.
+7. If we need to create a fix from feedbacks, then this cycle repeats.
+8. Otherwise, follow the [stable release process](./RELEASING.md#Release)
+
+## Release
+
 1. Update the `version` in [gradle.properties](gradle.properties) to a non-`SNAPSHOT`.
 2. Update [CHANGELOG.md](CHANGELOG.md)
-  - Add the new version section and move the *unreleased changes* into it.
-  - Update the links at the end of the page.
+- Add the new version section and move the *unreleased changes* into it.
+- Update the links at the end of the page.
```

Comment:
Don't trust this 🐰 indentation is required (and you need 3 spaces here)
```suggestion
   - Add the new version section and move the *unreleased changes* into it.
   - Update the links at the end of the page.
```

Reply:
Fixed in [78a11a2](https://github.com/adevinta/spark-android/pull/1185/commits/78a11a24231d904d83dd69ae74eee3d6d1d67ae0)